landingzones/tests/test_views_ajax.py

```diff
@@ -26,3 +26,16 @@ def test_post(self):
             }
         }
         self.assertEquals(response.data, expected)
+
+    def test_post_no_landingzone(self):
```

Comment:
Minor style point, `test_post_no_zone()` would probably be more consistent with the naming of other tests. Otherwise this looks good!